cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -192,12 +192,13 @@
     "BaseOS": {
       "Description": "Base OS type for cluster AMI",
       "Type": "String",
-      "Default": "alinux",
+      "Default": "alinux2",
```

Comment:
Avoiding switching the default

Reply:
Yeah agreed. I will do so, just haven't gotten around to making the changes yet.